src/components/dropdowns/dropdowns.html

```diff
@@ -0,0 +1,61 @@
+<section class="docs-block">
+    <aside class="docs-block__info">
+        <h3 class="docs-block__header">Dropdowns</h3>
+    </aside>
+    <div class="docs-block__content">
+        <div class="mint-dropdown">
+            <div class="mint-dropdown__label">All options</div>
```

Comment:
@mr-mig @matzimowski I just checked labels, I would suggest to update `mint-label` font-family here to extrabold instead black? Its more readable, and doesn't have big impact ie in search page.
![d1a948ee96f6e12b7adb9f034e43b01e](https://cloud.githubusercontent.com/assets/1142999/9765933/dbc86362-5716-11e5-84dc-d1c8c9b07846.gif)